### PR TITLE
PHOENIX-6242 Remove remaining commons-logging references

### PR DESCRIPTION
**Pull request by stoty**

### CODE REVIEW

File: phoenix-core/src/it/java/org/apache/phoenix/end2end/MaxLookbackIT.java [hunk 1]
```diff
@@ -60,7 +60,7 @@
 
 @NeedsOwnMiniClusterTest
 public class MaxLookbackIT extends BaseUniqueNamesOwnClusterIT {
-    private static final Log LOG = LogFactory.getLog(MaxLookbackIT.class);
+    public static final Logger LOG = LoggerFactory.getLogger(MaxLookbackIT.class);
```
Comment (virajjasani) [inline phoenix-core/src/it/java/org/apache/phoenix/end2end/MaxLookbackIT.java:63]: nit: keep it private? applicable to all places.

**Review by virajjasani — COMMENTED** [review]
Except for keeping all loggers private, everything looks good. Confirmed locally that all places are covered.

File: phoenix-core/src/it/java/org/apache/phoenix/end2end/MaxLookbackIT.java (hunk 1 above)
Comment (stoty) [reply phoenix-core/src/it/java/org/apache/phoenix/end2end/MaxLookbackIT.java:63]: Of course. Thanks for catching this copy-paste error.

Comment (stoty) [reply phoenix-core/src/it/java/org/apache/phoenix/end2end/MaxLookbackIT.java]: Done

**Review by virajjasani — APPROVED** [review]
+1 (non-binding)

**Comment by stoty** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 10s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ 4.x Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 30s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |  11m 44s |  4.x passed  |
| +1 :green_heart: |  compile  |   1m 40s |  4.x passed  |
| +1 :green_heart: |  checkstyle  |   0m 57s |  4.x passed  |
| +1 :green_heart: |  javadoc  |   2m  9s |  4.x passed  |
| +0 :ok: |  spotbugs  |   5m 17s |  root in 4.x has 1004 extant spotbugs warnings.  |
| +0 :ok: |  spotbugs  |   3m 43s |  phoenix-core in 4.x has 950 extant spotbugs warnings.  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 18s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   6m 29s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 31s |  the patch passed  |
| +1 :green_heart: |  javac  |   1m 31s |  the patch passed  |
| -1 :x: |  checkstyle  |   0m 54s |  root: The patch generated 12 new + 1009 unchanged - 13 fixed = 1021 total (was 1022)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  1s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  javadoc  |   1m 51s |  the patch passed  |
| +1 :green_heart: |  spotbugs  |   7m 57s |  the patch passed  |
||| _ Other Tests _ |
| -1 :x: |  unit  | 190m 30s |  root in the patch failed.  |
| +1 :green_heart: |  asflicense  |   0m 58s |  The patch does not generate ASF License warnings.  |
|  |   | 240m 43s |   |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | phoenix.end2end.index.IndexCoprocIT |
|   | phoenix.end2end.index.MutableIndexIT |
|   | TEST-[RangeScanIT_2] |
|   | phoenix.end2end.RowValueConstructorOffsetIT |
|   | phoenix.end2end.IndexExtendedIT |
|   | phoenix.end2end.SequenceIT |
|   | phoenix.end2end.SkipScanQueryIT |
|   | phoenix.end2end.AlterTableIT |
|   | phoenix.end2end.index.GlobalImmutableTxIndexIT |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.40 ServerAPI=1.40 base: https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1001/2/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/phoenix/pull/1001 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs hbaseanti checkstyle compile xml |
| uname | Linux f4e874fb6b16 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev/phoenix-personality.sh |
| git revision | 4.x / 5e70f76 |
| Default Java | Private Build-1.8.0_242-8u242-b08-0ubuntu3~16.04-b08 |
| checkstyle | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1001/2/artifact/yetus-general-check/output/diff-checkstyle-root.txt |
| unit | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1001/2/artifact/yetus-general-check/output/patch-unit-root.txt |
|  Test Results | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1001/2/testReport/ |
| Max. process+thread count | 6065 (vs. ulimit of 30000) |
| modules | C: phoenix-core . U: . |
| Console output | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-1001/2/console |
| versions | git=2.7.4 maven=3.3.9 spotbugs=4.1.4 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Review by gjacoby126 — APPROVED** [review]
+1, thanks @stoty

**Comment by gjacoby126** [comment]
JIRA is marked resolved, closing PR.

**Pull request closed without merging**